Code/Legacy/CrySystem/DebugCallStack.cpp

```diff
@@ -666,114 +657,6 @@ void DebugCallStack::LogExceptionInfo(EXCEPTION_POINTERS* pex)
     }
 }
 
-
-INT_PTR CALLBACK DebugCallStack::ExceptionDialogProc(HWND hwndDlg, UINT message, WPARAM wParam, LPARAM lParam)
-{
-    static EXCEPTION_POINTERS* pex;
-
-    switch (message)
-    {
-    case WM_INITDIALOG:
```

Comment:
I hope there isn't any more Legacy Windows UI code :-)

Reply:
Seems like that was the last occurence :)